### PR TITLE
FR10.1 The system should allow verified users to show the taken micro-loans

### PR DESCRIPTION
**Pull request by kimoshinowo**
- FR10.1.1 The system should accept the command SHOWTAKENLOANS
- FR10.1.2 The system should display all taken loans with the loan number, customer name, amount, interest rate, and account taken by
- FR10.1.3 The system should display a meaningful error message if the customer has not any open loans
- Add SHOWTAKENLOANS to help command and test

### CODE REVIEW

File: test/NewBankTest.java [hunk 1]
```diff
@@ -540,4 +540,49 @@ public void insufficientFunds() {
...
+        String result = test.processRequest(clientId, "OFFERLOAN 500 66666666 365 5");
+
+        Assertions.assertEquals("Success. Loan of 500.0 offered from 66666666 for 365 days with interest of 5%", result);
+        Assertions.assertEquals(500, account.getAvailableBalance());
+
+        String result2 = test.processRequest(clientId2, "ACCEPTLOAN 1 77777777");
+
+        Assertions.assertEquals("Success. Loan number 1 accepted by account 77777777.", result2);
+
+        String result3 = test.processRequest(clientId, "SHOWTAKENLOANS");
```
Comment (Jabul99) [inline test/NewBankTest.java:569]: Here, the client2 should be taken.

File: newbank/server/NewBank.java [hunk 2]
```diff
@@ -158,6 +161,20 @@ public synchronized String processRequest(CustomerID customer, String request) {
         return "FAIL";
     }
 
+    private String showTakenLoans(CustomerID customerID) {
+        Customer customer = this.getCustomer(customerID);
+        String result = "";
+
+        for (Account account : customer.getAllAccounts()) {
+            for(Loan loan: loans){
+                if(loan.getAccountFrom().getAccountNumber().equals(account.getAccountNumber())){
```
[the remark this answers is not in the document]
Comment (kimoshinowo) [reply newbank/server/NewBank.java:170]: Yes it should, Good spot! I missed that bit!

File: newbank/server/NewBank.java [hunk 3]
```diff
@@ -158,6 +161,20 @@ public synchronized String processRequest(CustomerID customer, String request) {
         return "FAIL";
     }
 
+    private String showTakenLoans(CustomerID customerID) {
+        Customer customer = this.getCustomer(customerID);
+        String result = "";
+
+        for (Account account : customer.getAllAccounts()) {
+            for(Loan loan: loans){
+                if(loan.getAccountTo().getAccountNumber().equals(account.getAccountNumber())){
```
Comment (Jabul99) [inline newbank/server/NewBank.java:170]: The accountto could be null, if the loan is not taken. The rest is fine.

**Comment by Rubberduck1** [comment]
Looks great, thanks for the good work @kimoshinowo @Jabul99

**Comment by kimoshinowo** [comment]
[FR10.1 The system should allow verified users to show the taken micro-loans](https://trello.com/c/7dWTNfm0/44-fr101-the-system-should-allow-verified-users-to-show-the-taken-micro-loans)

**Pull request merged**